### DISCUSSION TITLE
Link to issue hidden comment works in Chrome but not in Firefox

### DISCUSSION THREAD

**Discussion by fantasai** [community/community#56142] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Opening this link https://github.com/w3c/csswg-drafts/issues/7589#issuecomment-1380842759 which goes to a currently-hidden comment (because of so many comments) doesn't seem to work in Firefox. It works in Chrome, though. It seems that the GitHub team did not test in Firefox.

Please adopt a policy of making sure features work in Firefox. I suggest having at least one member of your team use Firefox as their main development browser, so that you notice things like this (and the obnoxious popup bug that made GitHub barely usable in Firefox for weeks).

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by airtower-luna** [comment, marked as answer]
That link works fine for me in Firefox 113. Which version are you using? :fox_face:
